### PR TITLE
Jakub refactor styling 237

### LINKED ISSUE #237: Consistently use one technique to add styling
We have a mix of inline styles, CSS and Sass modules. We should pick one approach and ensure the project consistently uses it. In particular, we should (probably) get rid of inline styles.

**Implementation Plan:**

- [x] use PascalCase everywhere to name components files
- [x] import scss modules as `styles` everywhere
- [x] use camelCase when defining scss modules classes
- [x] move inline styles to scss modules
- [x] use @use to import sass modules instead of @import 
- [x] remove app.global.css
- [x] refactor global scss modules

### PR DESCRIPTION
**Pull request by jchojna**
Closes #237 

### Description
- applied PascalCase everywhere to name components files
- imported scss modules as styles everywhere
- used camelCase when defining scss modules classes
- moved inline styles to scss modules
- used @use to import sass modules instead of @import
- removed app.global.css
- refactored global scss modules

### How to test
- check code structure, quality etc.
- click through the app to detect if there's no missing styles/mismatches etc

### CODE REVIEW

**Comment by jchojna** [comment]
I had to install `sass` dependency to be able to use `@use` functionality. Besides `node-sass` used so far is deprecated. After the app's all dependencies upgrade, I guess there won't be node-sass anymore, but sass installed as this is the best practice right now for compiling sass code.

File: declarations.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+declare module '*.module.scss';
```
Comment (kamilzyla) [inline declarations.d.ts:1]: Let's remove this and use a `modules.d.ts` file (see my other comments).

File: package.json [hunk 2]
```diff
@@ -262,6 +267,7 @@
     "prettier": "^1.19.1",
     "react-test-renderer": "^16.12.0",
     "rimraf": "^3.0.0",
+    "sass": "^1.53.0",
```
Comment (kamilzyla) [inline package.json:270]: Can we remove `node-sass` now?

File: tsconfig.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
     "resolveJsonModule": true,
     "allowJs": true
   },
+  "include": ["app", "./declarations.d.ts"],
```
Comment (kamilzyla) [inline tsconfig.json:25]: Shouldn't be necessary with `modules.d.ts` (see my other comments).

**Review by kamilzyla — CHANGES_REQUESTED** [review]
LGTM, great work, so much cleaner! :tada: :100: Only minor comments + we need to fix CI.

File: package.json (hunk 2 above)
Comment (jchojna) [reply package.json:270]: After `node-sass` removal I had some errors after cleaning the repo and `yarn`. I leave it as it is for now

File: app/index.tsx [hunk 4]
```diff
@@ -5,7 +5,6 @@ import i18n from 'i18next';
 import { initReactI18next } from 'react-i18next';
 
 import Root from './containers/Root';
-import './app.global.css';
 import './styles/app.global.scss';
```
[the remark this answers is not in the document]
Comment (jchojna) [reply app/index.tsx:8]: After only changing the name of the file to `global.css` and updating the corresponding path in `index.tsx`, there is some error in CI and the app looked like it misses some styling. I left it as `app.global.scss`

**Review by kamilzyla — APPROVED** [review]
LGTM :+1:

**Pull request merged**